robottelo/decorators.py

```diff
@@ -273,12 +274,18 @@ def bz_bug_is_open(bug_id):
     :rtype: bool
 
     """
+    upstream_mode = str(conf.properties.get('main.upstream', 1)) == '1'
```

Comment:
`upstream_mode = conf.properties.get('main.upstream', '1')) == '1'` will do the same. All properties are returned as string.